example-app/SantokuApp/.eslintrc.js

```diff
@@ -5,7 +5,8 @@ module.exports = {
   plugins: ['strict-dependencies'],
   root: true,
   // universe/native: https://github.com/expo/expo/tree/master/packages/eslint-config-universe
-  extends: ['universe/native', 'plugin:react-hooks/recommended'],
+  // eslint-comments/recommended: https://github.com/mysticatea/eslint-plugin-eslint-comments
```

Comment:
`recommended` の説明はドキュメントのrulesページのほうがわかりやすいと思いました。
> 🌟 mark: the rule which is enabled by eslint-comments/recommended preset.

```suggestion
  // eslint-comments/recommended: https://mysticatea.github.io/eslint-plugin-eslint-comments/rules/
```